### PR TITLE
Feature/room pictures

### PR DESCRIPTION
**Pull request by jmettler27**
Adds a list of photos to each classroom
Allows a user to see each photo in detail by clicking on it, and to zoom in or out.
For now the list of photos is not linked to Firebase Storage, so the list used is fake and is the same for all rooms.

### CODE REVIEW

**Review by emilio-gambino — APPROVED** [review]
Nothing to say, great !

File: app/src/androidTest/java/com/github/sdp/ratemyepfl/activity/ImageDetailActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+
+@HiltAndroidTest
+class ImageDetailActivityTest {
```
Comment (mbouilloud) [inline app/src/androidTest/java/com/github/sdp/ratemyepfl/activity/ImageDetailActivityTest.kt:22]: https://stackoverflow.com/questions/11523423/how-to-generate-zoom-pinch-gesture-for-testing-for-android, you can maybe use this to fully test the activity.

File: app/src/main/java/com/github/sdp/ratemyepfl/database/ReviewsRepository.kt [hunk 2]
```diff
@@ -6,4 +6,5 @@ interface ReviewsRepository {
     fun add(value: Review)
     suspend fun get(): List<Review>
     suspend fun getByReviewableId(id: String?): List<Review>
+    suspend fun getPhotosByReviewableId(id: String?): List<Int>
```
Comment (mbouilloud) [inline app/src/main/java/com/github/sdp/ratemyepfl/database/ReviewsRepository.kt:9]: I don't think this method should be here, we should have a separate Repository for pictures. Maybe I am missing something.

File: app/src/main/java/com/github/sdp/ratemyepfl/database/ReviewsRepositoryImpl.kt [hunk 3]
```diff
@@ -10,6 +11,16 @@ class ReviewsRepositoryImpl @Inject constructor() : ReviewsRepository, Repositor
 
     companion object {
         const val COLLECTION_PATH = "reviews"
+
+        // Fake photo ids
+        private val FAKE_PHOTOS = listOf(
```
Comment (mbouilloud) [inline app/src/main/java/com/github/sdp/ratemyepfl/database/ReviewsRepositoryImpl.kt:16]: Same comment.

**Review by NicolasMatekalo — APPROVED** [review]
Nothing much to say, great work and good effort for testing! :)

File: app/src/androidTest/java/com/github/sdp/ratemyepfl/activity/ImageDetailActivityTest.kt (hunk 1 above)
Comment (jmettler27) [reply app/src/androidTest/java/com/github/sdp/ratemyepfl/activity/ImageDetailActivityTest.kt:24]: Yes you're right. Thank you for the reference, it is really helpful.

File: app/src/main/java/com/github/sdp/ratemyepfl/database/ReviewsRepository.kt (hunk 2 above)
Comment (jmettler27) [reply app/src/main/java/com/github/sdp/ratemyepfl/database/ReviewsRepository.kt:9]: Mhh yes, makes sense, as not all Reviewable have photos. I will create a special Repository.

**Review by mbouilloud — APPROVED** [review]

**Pull request merged**